docs/reference/glossary.md

```diff
@@ -93,6 +93,27 @@ develop, ship, and run applications
 - The docker daemon process running on the host which manages images and containers
 
 
+## Docker for Mac
+
+[Docker for Mac](https://docs.docker.com/docker-for-mac/) is an easy-to-install,
+lightweight Docker development environment designed specifically for the Mac. A
+native Mac application, Docker for Mac uses the macOS Hypervisor framework,
+networking, and filesystem. It's the best solution if you want to build, debug,
+test, package, and ship Dockerized applications on a Mac. Docker for Mac
+supersedes [Docker Toolbox](#toolbox) as state-of-the-art Docker for macOS.
+
+## Docker for Windows
+
+[Docker for Windows](https://docs.docker.com/docker-for-windows/) is an
+easy-to-install, lightweight Docker development environment designed
+specifically for Windows systems. Docker for Windows uses Microsoft Hyper-V, and
+runs as a native Windows app. It works with Windows Server 2016, and gives you
+the ability to set up and run Windows containers as well as the standard Linux
+containers (with an option to switch between the two). Docker for Windows is the
+best solution if you want to build, debug, test, package, and ship Dockerized
+applications on Windows machines. It supersedes [Docker Toolbox](#toolbox) as
+state-of-the-art Docker on Windows.
```

Comment:
@mstanleyjones see my d4mac comment, I'd like to keep as-is. I think "state-of-the-art" conveys something more compelling than "better option" and telegraphs more info in a condensed way.